source/PlanetPanel.cpp

```diff
@@ -90,6 +90,33 @@ void PlanetPanel::Step()
 		else
 			player.HandleBlockedMissions(Mission::LANDING, GetUI());
 	}
+
+	if(planet.HasShipyard())
+	{
+		int repairedFighters = 0;
+		for(const auto &it : player.Ships())
+		{
+			int repairSelf = it->DoRepairWreckedFighter();
+			if(repairSelf == 0)
+				repairedFighters += it->DoRepairMyWreckedFighters();
+			else
+				repairedFighters += repairSelf;
+		}
+
+		if(repairedFighters > 0)
+		{
+			ostringstream out;
+
+			string fighterWord = "fighters";
+			if(repairedFighters == 1)
+			{
+				fighterWord = "fighter";
+			}
+			out << "You quickly see your wrecked " << fighterWord << " transported to the shipyard for repair.";
```

Comment:
It's a minor thing, but it'd be nice if this correctly identified whether it was fighters, drones or both that had been repaired, and updated the text accordingly.

Reply:
An excellent point. 

What's the most canonical way to tell? Should it test for whether each ship that got repaired has any max crew? Should it use the `category`? Or is there a better way?

Reply:
We dont have much code linked to the names of the category yet,do we?
I'd check for crew and use "unmanned drone" and fighter, but mb its hacky

Reply:
In this case, I think just matching `category` is probably the best approach. If you report the name of the category directly, you would also have some degree of compatibility with categories added by plugins, though I don't think there'd be a way to support nonstandard pluralisations.